### ISSUE TITLE
The command `virt-install` run failed

### ISSUE THREAD

**Issue by imichael2e2** [coreos/fedora-coreos-docs#489]
When running commands in `getting-started-libvirt.adoc`, several
errors are reported for `virt-install`:

```
Starting install...
Allocating 'fcos-test-02-2.qcow2'                                         |    0 B  00:00:00 ... 
Removing disk 'fcos-test-02-2.qcow2'                                      |    0 B  00:00:00     
ERROR    internal error: process exited while connecting to monitor: 2022-11-22T02:13:17.404468Z qemu-kvm: on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny: Could not open 'on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny': No such file or directory
Domain installation does not appear to have been successful.
```

With debug info, bash shows the command being run is
```
+ virt-install --connect=qemu:///system --name=fcos-test-02 --vcpus=1 --memory=1024 --os-variant=fedora-coreos-stable --import --graphics=none --disk=size=10,backing_store=/path/to/fedora-coreos-36.20221030.3.0-qemu.x86_64.qcow2 --network bridge=virbr0 --qemu-commandline=-fw_cfg
```
The `-fw_cfg blabla` is just left half, another part is missing.

It seems like the common quoting issue for Bash, so I tried to escape
the quotes like

```
IGNITION_DEVICE_ARG="--qemu-commandline=\"-fw_cfg name=opt/com.coreos/config,file=${IGNITION_CONFIG}\"" 
```

But another error got:

```
Starting install...
Allocating 'fcos-test-02-2.qcow2'                                         |    0 B  00:00:00 ... 
Removing disk 'fcos-test-02-2.qcow2'                                      |    0 B  00:00:00     
ERROR    internal error: process exited while connecting to monitor: qemu-kvm: -fw_cfg name=opt/com.coreos/config,file=/path/to/mycoreos.ign: invalid option
Domain installation does not appear to have been successful.
```

With debug info, Bash shows the command being run is:
```
+ virt-install --connect=qemu:///system --name=fcos-test-02 --vcpus=1 --memory=1024 --os-variant=fedora-coreos-stable --import --graphics=none --disk=size=10,backing_store=/path/to/fedora-coreos-36.20221030.3.0-qemu.x86_64.qcow2 --network bridge=virbr0 '--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/mycoreos.ign"'
```

**Comment by travier** [comment]
I think your fix in https://github.com/imichael2e2/fedora-coreos-docs/commit/1c810858a5be9b47ceee12f2eda7ca845bf0bc20 looks good. Can you make a PR with that?

**Comment by imichael2e2** [comment]
> I think your fix in [imichael2e2@1c81085](https://github.com/imichael2e2/fedora-coreos-docs/commit/1c810858a5be9b47ceee12f2eda7ca845bf0bc20) looks good. Can you make a PR with that?

Thanks for the reply, in fact I had made a PR for this issue several days ago, and it is now in pending status and needs a review for the merge.  Please let me know if any changes need to be refined!  https://github.com/coreos/fedora-coreos-docs/pull/490

**Comment by jlebon** [comment]
#490 is merged, thanks!
